apps/spectator/components/Panel/Panel.css.ts

```diff
@@ -1,27 +1,31 @@
-import { theme } from '@hcc/styles';
-import { styleVariants } from '@vanilla-extract/css';
+import { rem, theme } from '@hcc/styles';
+import { style, styleVariants } from '@vanilla-extract/css';
+
+const panelItemBase = style({
+  ...theme.textVariants.default,
+  textAlign: 'center',
+  paddingBlock: rem(16),
```

Comment:
```suggestion
  paddingBlock: rem(12),
```

변경하는 과정에서 높이가 달라진 거 같습니다.

Reply:
만드는 과정에서 12px이 오히려 와이어프레임보다 얇쌍해진다 생각해서 16px로 임의로 조정한 것인데, 지금 다시보니 12px로 하는게 와이어프레임이랑 동일해지는 것 같습니다 임의로 고쳐서 혼선 드린 점 사과드립니다🙇‍♂️